### PR TITLE
docs: sample for creation of kyma env instance

### PR DESCRIPTION
**Pull request by lechnerc77**
## Purpose

* Enhancement of documentation based on community contribution

## Does this introduce a breaking change?

```
[ ] Yes
[X] No
```

## Pull Request Type

What kind of change does this Pull Request introduce?

```
[ ] Bugfix
[ ] Feature
[ ] Refactoring (no functional changes, no api changes)
[X] Documentation content changes
[ ] Other... Please describe:
```

## How to Test

n/a

## What to Check

* Check generated documentation

## Other Information

based on PR #429 

## Checklist for reviewer

The following organizational tasks must be completed before merging this PR:

* [X] The PR is assigned to the Terraform project and a status is set (typically "in review").
* [X] The PR has the matching labels assigned to it.
* [X] The PR has a milestone assigned to it.
* [X] If the PR closes an issue, the issue is referenced.
* [X] Possible follow-up items are created and linked.

### CODE REVIEW

File: examples/resources/btp_subaccount_environment_instance/resource.tf [hunk 1]
```diff
@@ -35,3 +35,26 @@ resource "btp_subaccount_environment_instance" "cloudfoundry" {
     delete = "30m"
   }
 }
+
+# creates a kyma environment in a given account
+# NOTE: for the available parameter values, check https://help.sap.com/docs/btp/sap-business-technology-platform/available-plans-in-kyma-environment
+resource "btp_subaccount_environment_instance" "kyma" {
+  subaccount_id    = "6aa64c2f-38c1-49a9-b2e8-cf9fea769b7f"
```
Comment (kuntzed) [inline examples/resources/btp_subaccount_environment_instance/resource.tf:42]: is this a real subaccount id? or was it made up for the documentation?

File: examples/resources/btp_subaccount_environment_instance/resource.tf [hunk 2]
```diff
@@ -35,3 +35,26 @@ resource "btp_subaccount_environment_instance" "cloudfoundry" {
...
+  name             = "my-kyma-environment"
+  environment_type = "kyma"
+  service_name     = "kymaruntime"
+  plan_name        = "aws"
+
+  parameters = jsonencode({
+    name            = "my-kyma-environment"
```
Comment (kuntzed) [inline examples/resources/btp_subaccount_environment_instance/resource.tf:49]: do you need to specify the name twice?

File: examples/resources/btp_subaccount_environment_instance/resource.tf (hunk 1 above)
Comment (lechnerc77) [reply examples/resources/btp_subaccount_environment_instance/resource.tf:42]: Made up for docu

File: examples/resources/btp_subaccount_environment_instance/resource.tf (hunk 2 above)
Comment (lechnerc77) [reply examples/resources/btp_subaccount_environment_instance/resource.tf:49]: According to the docu it is display-name and cluster name. They can differ.

**Review by kuntzed — APPROVED** [review]

**Pull request merged**